src/components/Header.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import logo from '../assets/planet.png';
+
+const Header = () => (
+  <header>
+    <div className="logo-container">
+      <img src={logo} alt="Space Travelers' Hub Logo" />
+      <h1>Space Travelers&apos; Hub</h1>
+    </div>
+    <nav>
+      <ul>
+        <li><Link to="/">Rockets</Link></li>
+        <li><Link to="/missions">Missions</Link></li>
+        <li><Link to="/profile">My Profile</Link></li>
```

Comment:
- [ ] From the requirement, you should use <NavLink>, not <Link> for your header links. NavLink has the benefit of allowing us style the selected menu option, which is also a requirement.